robotoff/spellcheck/patterns/__init__.py

```diff
@@ -0,0 +1,38 @@
+from typing import Dict
+from pathlib import Path
+
+from robotoff.spellcheck.base_spellchecker import BaseSpellchecker
+
+Patterns = Dict[str, str]
+
+
+class PatternsSpellchecker(BaseSpellchecker):
+
+    PATTERNS_PATHS = {"fr": Path(__file__).parent / "patterns_fr.txt"}
```

Comment:
The rest of the data file are in the data directory, in the project root. The path to the file is then added in settings.py. Can you do the same here?

Reply:
Indeed not a proper way to do it. 
I corrected it in https://github.com/openfoodfacts/robotoff/pull/181/commits/a93bd4ed70c72e2b78070a17db0fdad967da8362 .